tests/ports/psoc6/run_psoc6_tests.sh

```diff
@@ -188,7 +189,14 @@ no_ext_hw_tests() {
 
 hw_ext_tests() {
   run_tests "hardware extended" ${dev_test} "${tests_psoc6_dir}/hw_ext" \
-  "-e ${tests_psoc6_dir}/hw_ext/i2c.py -e ${tests_psoc6_dir}/hw_ext/sdcard.py -e ${tests_psoc6_dir}/hw_ext/uart.py"
+  "-e ${tests_psoc6_dir}/hw_ext/i2c.py \
```

Comment:
Why do we exclude these tests from hw_ext? Ideally then only pin/pwm/signal are running as part of hw_ext? Actually, it would be eventually nice to have the chance to call each test individually as well.

Reply:
This needs to be changed... The problem is that initially all those hw extension were located in a single board.
But as we have added more tests, the hw extensions for each of those is spread across the different boards. This hw-ext clustering is not any longer very applicable as it is.

In the future, the new script should for a given board and hw_ext version, run all its supported tests, first single, then multi.